common/pom.xml

```diff
@@ -98,26 +100,49 @@
       <artifactId>antlr-runtime</artifactId>
       <version>3.4</version>
     </dependency>
-
     <dependency>
-        <groupId>joda-time</groupId>
-        <artifactId>joda-time</artifactId>
-        <version>2.3</version>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.3</version>
     </dependency>
 
+    <!-- Test dependencies -->
+    <dependency>
+      <groupId>com.carrotsearch.randomizedtesting</groupId>
```

Comment:
what motivated this change ? what are the benefits of randomizedtesting vs surefire plugin ?